### PR TITLE
Replaces 'lsof' with 'fuser'

### LINKED ISSUE #131: Newly introduced usage of lsof breaks container startup
**Description**

With merge request #130 you introduced the usage of `lsof` within the container. Unfortunately, this breaks the startup of the container in my runtime. The tool takes very long time to terminate in a container and furthermore its output is not applicable to detect, whether `mysqld` still has database files opened.

**Steps to reproduce the issue:**

1. Use Bitnami MySQL chart to deploy MySQL in kind running in Docker.

**Describe the results you received:**

Container is stuck running `lsof`, which takes a long time to terminate, and its output makes the check fail, whether `mysqld` still has database files opened:

[container.log](https://github.com/bitnami/bitnami-docker-mysql/files/6553298/container.log)

**Describe the results you expected:**

As `lsof` obviously does not work correctly due to container security, another mechanism should be used to detect still opened database files.

**Additional information you deem important (e.g. issue happens only occasionally):**

I am running
- Manjaro Linux
- kernel 5.12.2
- Docker 20.10.6
- kind 0.11.0
- Helm 3.5.3
- Bitnami MySQL Chart 8.5.9

The same happens when using podman 3.1.2 instead of Docker.

**Version**

- Output of `docker version`:

```
Client:
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.16.3
 Git commit:        370c28948e
 Built:             Mon Apr 12 14:10:41 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.6
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.3
  Git commit:       8728dd246c
  Built:            Mon Apr 12 14:10:25 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.5.0
  GitCommit:        8c906ff108ac28da23f69cc7b74f8e7a470d1df0.m
 runc:
  Version:          1.0.0-rc94
  GitCommit:        2c7861bc5e1b3e756392236553ec14a78a09f8bf
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

- Output of `docker info`:

```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.5.1-tp-docker)

Server:
 Containers: 1
  Running: 1
  Paused: 0
  Stopped: 0
 Images: 8
 Server Version: 20.10.6
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: false
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8c906ff108ac28da23f69cc7b74f8e7a470d1df0.m
 runc version: 2c7861bc5e1b3e756392236553ec14a78a09f8bf
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.12.2-1-MANJARO
 Operating System: Manjaro Linux
 OSType: linux
 Architecture: x86_64
 CPUs: 24
 Total Memory: 31.28GiB
 Name: konstanze
 ID: FPUJ:6YOK:CALA:JQJL:X6BN:OXG7:3KB7:BGFJ:K5MF:U47Z:AJXK:W3YG
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

The same happens when disabling cgroups v1 and just using cgroups v2.

### PR DESCRIPTION
**Pull request by randradas**
**Description of the change**

- Install a new package called 'psmisc' which contain the 'fuser' binary.
- Replaces _lsof_ with _fuser_. _fuser_ unix command is used to show which processes are using a specified computer file, file system, or Unix socket. According to findings and documentation _fuser_ fits perfectly when want to find process using specific files, however _lsof_ fits better when want to find the opposite, files opened by specific process.

**Benefits**

_fuser_ performs the same task that _lsof_ but it seems it does not take too long so the _mysql_stop_ function works as expected when waiting mysqld files to be unlocked.

**Possible drawbacks**

none

**Applicable issues**

- fixes #131 

**Additional information**

### CODE REVIEW

File: 5.7/debian-10/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ENV HOME="/" \
 
 COPY prebuildfs /
 # Install required system packages and dependencies
-RUN install_packages acl ca-certificates curl gzip libaio1 libatomic1 libc6 libgcc1 libncurses6 libsasl2-2 libssl1.1 libstdc++6 libtinfo6 lsof procps tar
+RUN install_packages acl ca-certificates curl gzip libaio1 libatomic1 libc6 libgcc1 libncurses6 libsasl2-2 libssl1.1 libstdc++6 libtinfo6 lsof procps tar psmisc
```
Comment (juan131) [inline 5.7/debian-10/Dockerfile:11]: Should we remove 'lsof' from the list? It's not required anymore, right?

File: 8.0/debian-10/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ ENV HOME="/" \
 
 COPY prebuildfs /
 # Install required system packages and dependencies
-RUN install_packages acl ca-certificates curl gzip libaio1 libc6 libcom-err2 libgcc1 libgssapi-krb5-2 libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 libncurses6 libsasl2-2 libssl1.1 libstdc++6 libtinfo6 lsof procps tar
+RUN install_packages acl ca-certificates curl gzip libaio1 libc6 libcom-err2 libgcc1 libgssapi-krb5-2 libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 libncurses6 libsasl2-2 libssl1.1 libstdc++6 libtinfo6 lsof procps tar psmisc
```
Comment (juan131) [inline 8.0/debian-10/Dockerfile:11]: Same concern here

**Review by juan131 — COMMENTED** [review]
Thanks for this fix! Please check my comments

File: 5.7/debian-10/Dockerfile (hunk 1 above)
Comment (randradas) [reply 5.7/debian-10/Dockerfile:11]: oh, you are completely right, I fix it.

File: 8.0/debian-10/Dockerfile (hunk 2 above)
Comment (randradas) [reply 8.0/debian-10/Dockerfile:11]: I fix it

**Review by juan131 — APPROVED** [review]
LGTM! Please don't merge the changes until they're replicated in our internal system.

**Pull request merged**